drv/sidecar-front-io/sidecar_qsfp_x32_controller.adoc

```diff
@@ -320,7 +320,7 @@ STATUS_PORT0
 | Bits | SW Access | Name | Function
 |[7:5] |  | - | Reserved
 |[4] | r | BUSY | '1' if the bus is busy.
-|[3:0] | r | ERROR | 0x0 = NoError, 0x01 = NoModule, 0x02 = NoPower, 0x03 = PowerFault, 0x04 = I2cAddressnack, 0x05 = I2cByteNack
```

Comment:
Are we intentionally removing the value list from the docs?  I often find it helpful!

Reply:
Interesting, they remain in the HTML page, so I guess it's fine?

Reply:
Yeah, thanks to Nathanael we actually have a mechanism to embed enums into our register descriptions which are then used to generate both BSV and Rust types! It'll also keep the docs in sync with the code, so its a win all around. Here's where that exists: https://github.com/oxidecomputer/quartz/blob/589697de44b7e173ad32123a7ecabb81703ea9e4/hdl/boards/sidecar/qsfp_x32/QSFPModule/qsfp_modules_top.rdl#L113-L121